src/core/engines/Cesium/Feature/Raster/hooks.ts

```diff
@@ -10,6 +10,8 @@ import md5 from "js-md5";
 import { useEffect, useMemo, useRef } from "react";
 import { useCesium } from "resium";
 
+import { clearAllExpressionCaches } from "@reearth/core/mantle/evaluator/simple";
```

Comment:
```suggestion
import { clearAllExpressionCaches } from "../../..";
```

Re-export clearAllExpressionCaches at `../../..`